### PR TITLE
Add description for capacities (and muted texts in setup)

### PR DESCRIPTION
**Pull request by ccailly**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 

Add a description to each capacity.

image

### CODE REVIEW

**Review by orthagh — DISMISSED** [review]
Regarding design : almost ok, I still think we loose a big space in the middle. I don't want to remove it entirely, but curently there is more space dedicated to emptyness than actual content. Maybe we can inverse space usage.

Another thing, if description is not provided, or empty, the whole block should not be displayed.

Regarding descriptions content, well, they don't inform about anything for the moment 😬 
I mean, yeah, Appliance capacity allows indeed to manage Appliances for the assets.
Don't take this as a personal criticism, but maybe another person, more experienced about the application could take a shot about the content.

File: src/Asset/Capacity/HasAntivirusCapacity.php [hunk 1]
```diff
@@ -51,6 +51,11 @@ public function getSearchOptions(string $classname): array
         return ItemAntivirus::rawSearchOptionsToAdd();
     }
 
+    public function getDescription(): string
```
Comment (AdrienClairembault) [inline src/Asset/Capacity/HasAntivirusCapacity.php:54]: A general remark here, we merged last week the PHP 8.3 polyfill, which mean we should try to use the `#[Override]` data attribute every time we override a method.

See https://stitcher.io/blog/override-in-php-83

**Review by AdrienClairembault — CHANGES_REQUESTED** [review]

**Comment by ccailly** [comment]
This is what it looks like with and without description. Regarding the content of the descriptions, I would like help in producing relevant descriptions.
image

**Comment by orthagh** [comment]
Seems good ! The design doesn't suffer when a capacity is without description

**Review by AdrienClairembault — APPROVED** [review]

**Comment by AdrienClairembault** [comment]
What the next step for this ?
Should it be merged as it is and then correct texts can be added later on or do you want to wait for the updates texts before merging ?

**Comment by cedric-anne** [comment]
IMHO, if only a few capacities are going to have a description, it would probably be more relevant to just have a :grey_question: icon that opens a tooltip. Indeed, descriptions may b far longer than only 2 lines and in this case the current design would probably be not adapted.

**Comment by AdrienClairembault** [comment]
To move forward, maybe it can be merged as it is for now (as it isn't really a ground breaking feature) ?

Then we can re-evaluate it later on when all capacities are finished and see how many capacities need description and how do we want to display them.

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**